setup.py

```diff
@@ -17,7 +17,8 @@
           'PyGithub',
           'PyYAML',
           'dateutils',
-          'python-gitlab'
+          'python-gitlab',
+          'mock'
```

Comment:
Ah, instead of in the ``install_requires=...`` list, you should put this in a ``tests_require=...`` list.